Core/Src/spi_command_handler.c

```diff
@@ -224,6 +224,264 @@ void spi_transfer_image() {
     static uint8_t image_data[IRIS_IMAGE_TRANSFER_BLOCK_SIZE];
     static uint16_t num_transfers;
 
+    uint8_t image_data_buffer[4574] = {
```

Comment:
Is it important to have test data checked in? Personally, I like generating self-describing (equals self-checking) data dynamically.